tests/test_message/event_sample.py

```diff
@@ -0,0 +1,204 @@
+    "collaborators_url": "https://api.github.com/repos/brngylni/demo_repo/collaborators{/collaborator}",
+    "teams_url": "https://api.github.com/repos/brngylni/demo_repo/teams",
+    "hooks_url": "https://api.github.com/repos/brngylni/demo_repo/hooks",
+    "issue_events_url": "https://api.github.com/repos/brngylni/demo_repo/issues/events{/number}",
+    "events_url": "https://api.github.com/repos/brngylni/demo_repo/events",
+    "assignees_url": "https://api.github.com/repos/brngylni/demo_repo/assignees{/user}",
+    "branches_url": "https://api.github.com/repos/brngylni/demo_repo/branches{/branch}",
+    "tags_url": "https://api.github.com/repos/brngylni/demo_repo/tags",
+    "blobs_url": "https://api.github.com/repos/brngylni/demo_repo/git/blobs{/sha}",
+    "git_tags_url": "https://api.github.com/repos/brngylni/demo_repo/git/tags{/sha}",
+    "git_refs_url": "https://api.github.com/repos/brngylni/demo_repo/git/refs{/sha}",
+    "trees_url": "https://api.github.com/repos/brngylni/demo_repo/git/trees{/sha}",
+    "statuses_url": "https://api.github.com/repos/brngylni/demo_repo/statuses/{sha}",
+    "languages_url": "https://api.github.com/repos/brngylni/demo_repo/languages",
+    "stargazers_url": "https://api.github.com/repos/brngylni/demo_repo/stargazers",
+    "contributors_url": "https://api.github.com/repos/brngylni/demo_repo/contributors",
+    "subscribers_url": "https://api.github.com/repos/brngylni/demo_repo/subscribers",
+    "subscription_url": "https://api.github.com/repos/brngylni/demo_repo/subscription",
+    "commits_url": "https://api.github.com/repos/brngylni/demo_repo/commits{/sha}",
+    "git_commits_url": "https://api.github.com/repos/brngylni/demo_repo/git/commits{/sha}",
+    "comments_url": "https://api.github.com/repos/brngylni/demo_repo/comments{/number}",
+    "issue_comment_url": "https://api.github.com/repos/brngylni/demo_repo/issues/comments{/number}",
+    "contents_url": "https://api.github.com/repos/brngylni/demo_repo/contents/{+path}",
+    "compare_url": "https://api.github.com/repos/brngylni/demo_repo/compare/{base}...{head}",
+    "merges_url": "https://api.github.com/repos/brngylni/demo_repo/merges",
+    "archive_url": "https://api.github.com/repos/brngylni/demo_repo/{archive_format}{/ref}",
+    "downloads_url": "https://api.github.com/repos/brngylni/demo_repo/downloads",
+    "issues_url": "https://api.github.com/repos/brngylni/demo_repo/issues{/number}",
+    "pulls_url": "https://api.github.com/repos/brngylni/demo_repo/pulls{/number}",
+    "milestones_url": "https://api.github.com/repos/brngylni/demo_repo/milestones{/number}",
+    "notifications_url": "https://api.github.com/repos/brngylni/demo_repo/notifications{?since,all,participating}",
+    "labels_url": "https://api.github.com/repos/brngylni/demo_repo/labels{/name}",
+    "releases_url": "https://api.github.com/repos/brngylni/demo_repo/releases{/id}",
+    "deployments_url": "https://api.github.com/repos/brngylni/demo_repo/deployments",
```

Comment:
Is there a way to emulate this without using a secondary external repository?

I want our test cases to be self-sufficient and not dependent on external factors.